codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -115,49 +108,4 @@ abstract class AWSHTTPBindingProtocolGenerator(
         operationMiddleware.appendMiddleware(operation, OperationEndpointResolverMiddleware(ctx))
         operationMiddleware.appendMiddleware(operation, UserAgentMiddleware(ctx.settings))
     }
-
-    fun outputStreamingShapes(ctx: ProtocolGenerator.GenerationContext): MutableSet<MemberShape> {
-        val streamingShapes = mutableMapOf<ShapeId, MemberShape>()
-        val streamingOperations = getHttpBindingOperations(ctx).filter { it.isOutputEventStream(ctx.model) }
-        streamingOperations.forEach { operation ->
-            val input = operation.output.get()
-            val streamingMember = ctx.model.expectShape(input).findStreamingMember(ctx.model)
-            streamingMember?.let {
-                val targetType = ctx.model.expectShape(it.target)
-                streamingShapes[targetType.id] = it
-            }
-        }
-
-        return streamingShapes.values.toMutableSet()
-    }
-
-    fun inputStreamingShapes(ctx: ProtocolGenerator.GenerationContext): MutableSet<UnionShape> {
-        val streamingShapes = mutableSetOf<UnionShape>()
-        val streamingOperations = getHttpBindingOperations(ctx).filter { it.isInputEventStream(ctx.model) }
-        streamingOperations.forEach { operation ->
-            val input = operation.input.get()
-            val streamingMember = ctx.model.expectShape(input).findStreamingMember(ctx.model)
-            streamingMember?.let {
-                val targetType = ctx.model.expectShape(it.target)
-                streamingShapes.add(targetType as UnionShape)
-            }
-        }
-        return streamingShapes
-    }
-
-    override fun generateMessageMarshallable(ctx: ProtocolGenerator.GenerationContext) {
-        var streamingShapes = inputStreamingShapes(ctx)
-        val messageMarshallableGenerator = MessageMarshallableGenerator(ctx, defaultContentType)
-        streamingShapes.forEach { streamingMember ->
-            messageMarshallableGenerator.render(streamingMember)
-        }
-    }
-
-    override fun generateMessageUnmarshallable(ctx: ProtocolGenerator.GenerationContext) {
-        var streamingShapes = outputStreamingShapes(ctx)
-        val messageUnmarshallableGenerator = MessageUnmarshallableGenerator(ctx)
-        streamingShapes.forEach { streamingMember ->
-            messageUnmarshallableGenerator.render(streamingMember)
-        }
-    }
```

Comment:
The two generator files below plus the event stream codegen tests and the supporting Smithy test fixture are all moved to smithy-swift.